### ISSUE TITLE
Create database

### ISSUE THREAD

**Issue by f0rever-johnson** [mevdschee/php-crud-api#376]
Thank you for your hard work!!  I just have a request. Is it possible to create a database from this script? Like POST /"DATABASE NAME" ? & then create a table on that database? essentially an endpoint to dynamically creating databases & tables.

**Comment by mevdschee** [comment]
>Is it possible to create a database from this script?

No, but it is not a bad idea. I'll register it as a feature request.

This might be added (at some point) to v2 as that version has more advanced meta data support.

**Comment by f0rever-johnson** [comment]
Thanks!! That would be awesome. Right now I’m writing stored procedures
with prepare statements in MySQL, & then calling them via the php-sp-api.
Do you think that is a viable solution?
-- 
Everaldlee Johnson

**Comment by mevdschee** [comment]
>Do you think that is a viable solution?

Sure it is.. :+1

**Comment by mevdschee** [comment]
@f0rever-johnson I'm aiming for such support in v2. I have reflection, table renames and column updates implemented currently. You can follow it here: https://github.com/mevdschee/php-crud-api2

**Comment by f0rever-johnson** [comment]
Hey, any update to dynamically creating databases/tables?

**Comment by mevdschee** [comment]
Hi Everaldlee, Thank you for asking this question. Creating tables is now possible, creating databases is not (yet) supported. I hope this helps you. Kind regards, Maurits

**Comment by f0rever-johnson** [comment]
I am not seeing anything regarding creating tables. Is there an example or a specify endpoint I should be using?

**Comment by mevdschee** [comment]
You need to enable the 'columns' controller, see also: https://github.com/mevdschee/php-crud-admin/blob/main/src/index.php

**Comment by f0rever-johnson** [comment]
I enabled the columns controller. However I am unsure on how to access the endpoint to create a table. Essentially I have a sample table(s) I would like to duplicate whenever a new user is created in my identityprovider.

**Comment by mevdschee** [comment]
> I have a sample table I would like to duplicate

This is not yet supported, although one could write this as middleware, executing certain queries when a new authenticated user is encountered.
